docs/getting-started/getting-started-with-wdl.rst

```diff
@@ -61,8 +65,10 @@ repository:
 ::
 
     task bamstats {
-        File bam_input
-        Int mem_gb
+        input {
```

Comment:
This docs refers to and link out to the existing bamstats wdl, but that link isn't WDL 1.0. Ideally both would be identical, but I'm not sure if a 1.0 version exists (either registered in dockstore or as a branch). I'm kind  of grappling with this on the CWL docs too.